src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,136 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private StorageDao storageDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUP() {
+        storageDao = new StorageDaoImpl();
+    }
+
+    @Test
+    void register_loginLessThanMinLength_notOk() {
+        User user = new User();
+        user.setLogin("Bob");
+        user.setPassword("ghjkl543");
+        user.setAge(23);
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void register_userAllInvalidFields_notOk() {
+        User user = new User();
+        user.setLogin("Micael");
+        user.setPassword("ghjkl");
+        user.setAge(15);
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(user);
```

Comment:
This test method name suggests that all user fields are invalid, but only the password length and age are not meeting the requirements. Additionally, it is better to separate tests for different invalid fields to ensure each validation is tested independently.